### PR TITLE
Keycloak docs fix wrong link and add small enhancement

### PR DESCRIPTION
**Pull request by jedla97**
Fix wrong link to Quarkus as the links return 404.

Fix wrong imports.

Add notes where to find the quarkus-realm.json and keycloak-keystore.jks with links to it.

### CODE REVIEW

File: docs/src/main/asciidoc/security-keycloak-authorization.adoc [hunk 1]
```diff
@@ -199,6 +199,8 @@ NOTE: Adding a `%prod.` profile prefix to `quarkus.oidc.auth-server-url` ensures
 
 NOTE: By default, applications using the `quarkus-oidc` extension are marked as a `service` type application (see `quarkus.oidc.application-type`). This extension also supports only  `web-app` type applications but only if the access token returned as part of the authorization code grant response is marked as a source of roles: `quarkus.oidc.roles.source=accesstoken` (`web-app` type applications check ID token roles by default).
 
+NOTE: The configurated `quarkus-realm.json` file for this example can be found in https://github.com/quarkusio/quarkus-quickstarts/blob/main/security-keycloak-authorization-quickstart/config/quarkus-realm.json[quarkus-quickstarts/security-keycloak-authorization-quickstart/config]
```
Comment (sberyozkin) [inline docs/src/main/asciidoc/security-keycloak-authorization.adoc:202]:
```suggestion
```

I'm not sure adding this note does not introduce duplication as just a few lines below there is 

Import the [realm configuration file](https://github.com/quarkusio/quarkus/blob/6dc006f7d86d6f9129997ca6263dd79f82213c79/docs/src/main/asciidoc/%7Bquickstarts-tree-url%7D/security-keycloak-authorization-quickstart/config/quarkus-realm.json)

File: docs/src/main/asciidoc/security-keycloak-authorization.adoc [hunk 2]
```diff
@@ -210,7 +212,7 @@ To start a Keycloak Server you can use Docker and just run the following command
 docker run --name keycloak -e KEYCLOAK_ADMIN=admin -e KEYCLOAK_ADMIN_PASSWORD=admin -p 8543:8443 -v "$(pwd)"/config/keycloak-keystore.jks:/etc/keycloak-keystore.jks quay.io/keycloak/keycloak:{keycloak.version} start  --hostname-strict=false --https-key-store-file=/etc/keycloak-keystore.jks
 ----
 
-where `keycloak.version` should be set to `17.0.0` or higher.
+where `keycloak.version` should be set to `23.0.0` or higher and the `keycloak-keystore.jks` can be found in https://github.com/quarkusio/quarkus-quickstarts/blob/main/security-keycloak-authorization-quickstart/config/keycloak-keystore.jks[quarkus-quickstarts/security-keycloak-authorization-quickstart/config]
```
Comment (sberyozkin) [inline docs/src/main/asciidoc/security-keycloak-authorization.adoc:215]:
```suggestion
where `keycloak.version` should be set to `23.0.0` or higher.
```

Users are not expected to use it somehow, the command above picks it up with `$(pwd)"/config/keycloak-keystore.jks`, so it is not clear what adding this information helps with, alongside a more important KC version update info

**Comment by sberyozkin** [comment]
@jedla97 Hi, thanks for your help here, I proposed to drop two notes regarding the location of resources only

File: docs/src/main/asciidoc/security-keycloak-authorization.adoc (hunk 2 above)
Comment (jedla97) [reply docs/src/main/asciidoc/security-keycloak-authorization.adoc:215]: My point in here is that if user not using quickstart but go through this by him self. He can easily miss this. I can drop it if you stand on this.

File: docs/src/main/asciidoc/security-keycloak-authorization.adoc (hunk 1 above)
Comment (jedla97) [reply docs/src/main/asciidoc/security-keycloak-authorization.adoc:202]: Ok, I'll drop this note

Comment (jedla97) [reply docs/src/main/asciidoc/security-keycloak-authorization.adoc]: Dropped

**Review by sberyozkin — APPROVED** [review]

File: docs/src/main/asciidoc/security-keycloak-authorization.adoc [hunk 3]
```diff
@@ -210,7 +210,7 @@ To start a Keycloak Server you can use Docker and just run the following command
...
```
Comment (michalvavrik) [inline docs/src/main/asciidoc/security-keycloak-authorization.adoc:213]: Please don't use direct links, use variables instead: `link:{quickstarts-blob-url}/security-keycloak-authorization-quickstart/config/keycloak-keystore.jks[text]`

**Pull request merged**